### PR TITLE
ORB 431 - Streamline contract events

### PR DESCRIPTION
**Pull request by RiccardoGalbusera**

### CODE REVIEW

File: contracts/Registry.sol [hunk 1]
```diff
@@ -12,6 +12,26 @@ contract Registry is IRegistry {
...
     bytes32[] public moduleKeys;
 
+    ///@notice emitted when governance address is changed
+    ///@param newGovernance the new governance address
+    event governanceChanged(address newGovernance);
+
+    ///@notice emitted when a contract is added to registry
+    ///@param newContract address of the new contract
+    ///@param moduleId keccak of module name
+    event newContractEvent(address newContract, bytes32 moduleId);
```
Comment (luduvigo) [inline contracts/Registry.sol:22]: Can you use the same convention as the other events, and so not use the word `Event` here, to keep it consistent?

File: contracts/Registry.sol [hunk 2]
```diff
@@ -30,6 +51,7 @@ contract Registry is IRegistry {
         require(modules[_id].contractAddress == address(0), 'Registry::addNewContract: Entry already exists.');
         modules[_id] = Entry({contractAddress: _contractAddress, activated: true});
         moduleKeys.push(_id);
+        emit newContractEvent(_contractAddress, _id);
```
Comment (luduvigo) [inline contracts/Registry.sol:54]: `newContractAdded` maybe

File: contracts/actions/AaveWithdraw.sol [hunk 3]
```diff
@@ -18,6 +18,12 @@ interface IAaveWithdraw {
 
 ///@notice action to withdraw tokens from aave protocol
 contract AaveWithdraw is IAaveWithdraw {
+    ///@notice emitted when a withdraw from aave is made
+    ///@param positionManager address of aave positionManager which withdrew
+    ///@param token token address
+    ///@param amount amount withdrawn
+    event AaveDepositEvent(address indexed positionManager, address token, uint256 amount);
```
Comment (luduvigo) [inline contracts/actions/AaveWithdraw.sol:25]: Can you check this one?

File: contracts/actions/ClosePosition.sol [hunk 4]
```diff
@@ -20,9 +20,9 @@ interface IClosePosition {
 
 contract ClosePosition is IClosePosition {
     ///@notice emitted when a UniswapNFT position is closed
-    ///@param from address of PositionManager
+    ///@param positionManager address of PositionManager
     ///@param tokenId Id of the closed token
-    event CloseUniPosition(address indexed from, uint256 tokenId);
+    event CloseUniPosition(address indexed positionManager, uint256 tokenId);
```
Comment (luduvigo) [inline contracts/actions/ClosePosition.sol:25]: Still here different, if we had the verbal form in the past, let's have it consistently on all the events

Comment (luduvigo) [reply contracts/actions/ClosePosition.sol:25]: or let's choose whichever convention that makes sense for all the events

File: contracts/actions/UpdateUncollectedFees.sol [hunk 5]
```diff
@@ -13,10 +12,11 @@ interface IUpdateUncollectedFees {
 
 contract UpdateUncollectedFees is IUpdateUncollectedFees {
     ///@notice emitted when a UniswapNFT position is updated
-    ///@param from address of PositionManager
+    ///@param positionManager address of PositionManager
+    ///@param tokenId Id of the updated position
     ///@param token0 fee collected
     ///@param token1 fee collected
-    event updateFees(address indexed from, uint256 token0, uint256 token1);
+    event updateFees(address indexed positionManager, uint256 tokenId, uint256 token0, uint256 token1);
```
Comment (luduvigo) [inline contracts/actions/UpdateUncollectedFees.sol:19]: 🥹

File: test/forkTests/actions/AaveDeposit.spec.ts [hunk 6]
```diff
@@ -168,7 +168,8 @@ describe('AaveDeposit.sol', function () {
 
       const events = (await tx.wait()).events;
       const depositEvent = events[events.length - 1];
-      const [id, shares] = abiCoder.decode(['uint256', 'uint256'], depositEvent.data);
+      console.log(depositEvent.data);
```
Comment (luduvigo) [inline test/forkTests/actions/AaveDeposit.spec.ts:171]: Should we keep this one?

**Review by luduvigo — CHANGES_REQUESTED** [review]

**Comment by RiccardoGalbusera** [comment]
Updated events to use past tense convention

**Review by luduvigo — APPROVED** [review]

**Pull request merged**